src/node.cc

```diff
@@ -3299,6 +3338,15 @@ void SetupProcessObject(Environment* env,
   env->SetMethod(process, "_setupPromises", SetupPromises);
   env->SetMethod(process, "_setupDomainUse", SetupDomainUse);
 
+#if HAVE_INSPECTOR
+  // process.inspector
+  if (use_inspector) {
```

Comment:
I would like to suggest moving this block (and 2 functions above) into inspector_agent.{h,cc} - this way there will be less conditional compilation to maintain.